src/main/java/com/emc/object/s3/jersey/ErrorFilter.java

```diff
@@ -55,10 +55,10 @@ public ClientResponse handle(ClientRequest request) throws ClientHandlerExceptio
 
             // check for clock skew (can save hours of troubleshooting)
             if (response.getStatus() == 403) {
-                Date clientTime = RestUtil.headerParse(RestUtil.getFirstAsString(request.getHeaders(), S3Constants.AMZ_DATE));
+                Date clientTime = RestUtil.headerParse(RestUtil.getFirstAsString(request.getHeaders(), S3Constants.AMZ_DATE), S3Constants.AMZ_DATE);
                 if (clientTime == null)
-                    clientTime = RestUtil.headerParse(RestUtil.getFirstAsString(request.getHeaders(), RestUtil.HEADER_DATE));
-                Date serverTime = RestUtil.headerParse(RestUtil.getFirstAsString(response.getHeaders(), RestUtil.HEADER_DATE));
+                    clientTime = RestUtil.headerParse(RestUtil.getFirstAsString(request.getHeaders(), RestUtil.HEADER_DATE), RestUtil.HEADER_DATE);
```

Comment:
seems like `HEADER_DATE` should **not** parse with `AMZ` format

Reply:
Refactored RestUtil.headerParse() method, removed the second parameter.